cmd/ci-operator-prowgen/main.go

```diff
@@ -112,7 +112,10 @@ func generatePodSpec(configFile, target string, additionalArgs ...string) *kubea
 func generatePodSpecTemplate(org, repo, configFile, release string, test *cioperatorapi.TestStepConfiguration, additionalArgs ...string) *kubeapi.PodSpec {
 	var template string
 	var clusterProfile cioperatorapi.ClusterProfile
+	var isUpgradeConfig bool
 	var needsReleaseRpms bool
+	var previousVersion string
```

Comment:
not entirely sure about those - should I copy config string here or just make `conf` visible later on?